manifests/candlepin.pp

```diff
@@ -0,0 +1,55 @@
+# Katello configuration for pulp
```

Comment:
I'll fix the typo, not sure about docs because I consider it a private class for now. The parameters are mostly there so you can unit test it without including katello (reducing test time)